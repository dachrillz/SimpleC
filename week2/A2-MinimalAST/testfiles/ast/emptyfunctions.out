Program
  List
    Function ID="a"
      List
      Block
        List
          Return
            Numeral NUMERAL="0"
    Function ID="b"
      List
      Block
        List
          Return
            Numeral NUMERAL="0"
