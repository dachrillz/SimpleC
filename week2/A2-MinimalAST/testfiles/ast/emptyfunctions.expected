Program
  List
    Function ID="a"
      List
        Return
    Function ID="b"
      List
        Return
